spec/lib/eventbrite_spec.rb

```diff
@@ -25,19 +25,20 @@
       end
 
       def stub_failed_eventbrite_request
-        stub_request(:get, eventbrite_api_url).to_return(body: { id: "123" })
-        stub_request(:get, eventbrite_api_url).to_return(status: 500)
+        stub_request(:get, eventbrite_api_url).to_return(
+          body: { id: "123" }.to_json,
+          status: 500
+        )
       end
     end
 
     def stub_successful_eventbrite_request
-      response = { events: [{ "title" => "event info" }] }
+      events = { events: [{ "title" => "event info" }] }
 
-      stub_request(:get, eventbrite_api_url).to_return(body: { id: "123" })
```

Comment:
`stub_request` with the same method and URL overwrites the stub each time, so this stub was never working